backend/src/app.ts

```diff
@@ -71,31 +71,44 @@ app.get('/review/:idType/:id', async (req, res) => {
   res.status(200).send(JSON.stringify(reviews));
 });
 
-app.get('/reviews/', async (req, res) => {
-  type ReqBody = { idType: string; ids: string[] };
-  const { idType, ids } = req.body as ReqBody;
-  const reviewsArr = await Promise.all(
-    ids.map(async (id) => {
-      const reviewDocs = (await reviewCollection.where(`${idType}`, '==', id).get()).docs;
-      const reviews: Review[] = reviewDocs.map((doc) => {
-        const { date, ...data } = doc.data();
-        return { date: date.toDate(), ...data } as Review;
-      });
-      return reviews;
-    })
-  );
-
-  const allReviews = reviewsArr.length > 1 ? reviewsArr : reviewsArr[0];
-
-  res.status(200).send(JSON.stringify(allReviews));
+app.get('/apts/:ids', async (req, res) => {
+  try {
+    const { ids } = req.params;
+    const idsList = ids.split(',');
+    const aptsArr = await Promise.all(
+      idsList.map(async (id) => {
+        const snapshot = await buildingsCollection.doc(id).get();
+        return { id, ...snapshot.data() } as ApartmentWithId;
+      })
+    );
+    console.log(aptsArr);
+    res.status(200).send(JSON.stringify(aptsArr));
+  } catch (err) {
+    res.status(400).send(err);
+  }
 });
 
-app.get('/apts/:id', async (req, res) => {
+app.get('/landlord/:id', async (req, res) => {
   try {
     const { id } = req.params;
-    const snapshot = await aptCollection.doc(id).get();
-    const aptDoc = { id, ...snapshot.data() } as ApartmentWithId;
-    res.status(200).send(JSON.stringify(aptDoc));
+    const ref = landlordCollection.doc(id);
+    const doc = await ref.get();
+    if (!doc.exists) {
+      throw new Error('Invalid id');
```

Comment:
since we're only running one instance of the server when we deploy it to prod (and not using separate containers or anything), it's a bit dangerous to throw errors since it'll result in downtime. maybe we can instead send an error status header in the response?